core/bentley/src/BeSQLite.ts

```diff
@@ -26,6 +26,29 @@ export enum DbOpcode {
   Update = 23,
 }
 
+/** Cause of conflict when applying a changeset
+ * @public
+ */
+export enum DbConflictCause {
```

Comment:
These two enums are only used in the private `onChangesetConflict` method. What's the motivation for making them `@public`?

Reply:
yes, that was a mistake. I have made them internal.
Also just made the following changes.
1. Move `onChangesetConflict()` to `BriefcaseDb` as it's the one that can apply changesets.
2. Update the existing test to make sure the `BriefcaseDb.onChangesetConflict` is called.